sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -532,6 +549,23 @@ void peekBatchMessagesFromSequence(MessagingEntityType entityType) {
             .verifyComplete();
     }
 
+    /**
+     * Verifies that an empty entity does not error when peeking.
+     */
+    @MethodSource("com.azure.messaging.servicebus.IntegrationTestBase#messagingEntityWithSessions")
+    @ParameterizedTest
+    void peekMessagesFromSequenceEmptyEntity(MessagingEntityType entityType, boolean isSessionEnabled) {
+        // Arrange
+        setSenderAndReceiver(entityType, TestUtils.USE_CASE_EMPTY_ENTITY, isSessionEnabled);
+
+        final int maxMessages = 10;
+        final int fromSequenceNumber = 1;
+
+        // Assert & Act
+        StepVerifier.create(receiver.peekMessagesAt(maxMessages, fromSequenceNumber))
```

Comment:
What is difference in this and test at line 382 above, it looks duplicate test or you wanted to test a different scenario ?
line 382 ` StepVerifier.create(receiver.peekMessagesAt(maxMessages, fromSequenceNumber))
            .verifyComplete();`

Reply:
The first one should be not plural (message). I'll fix it.